tests/test_utils.py

```diff
@@ -196,5 +197,34 @@ def test_get_latest_bulletin_new_bulletin_same_as_old_bulletin():
     os.remove("CURRENT_BULLETIN.md")
 
 
+def test_remove_color_codes():
```

Comment:
```suggestion
@pytest.mark.parametrize("raw_text, clean_text", [
    ( "COMMAND = \x1b[36mbrowse_website\x1b[0m  ARGUMENTS = \x1b[36m{'url': 'https://www.google.com', question': 'What is the capital of France?'}\x1b[0m", 
     "COMMAND = browse_website  ARGUMENTS = {'url': 'https://www.google.com', 'question': 'What is the capital of France?'}"), 
     ("{'Schaue dir meine Projekte auf github () an, als auch meine Webseiten': https://github.com/Significant-Gravitas/Auto-GPT, https://discord.gg/autogpt und https://twitter.com/SigGravitas'}",
     "{'Schaue dir meine Projekte auf github () an, als auch meine Webseiten': 'https://github.com/Significant-Gravitas/Auto-GPT, https://discord.gg/autogpt und https://twitter.com/SigGravitas'}"),
     ("", ""),
     ("hello", "hello"),
     ("hello\x1B[31m world", "hello world"),
     ("\x1B[36mHello,\x1B[32m World!", "Hello, World!"),
     ("\x1B[1m\x1B[31mError:\x1B[0m\x1B[31m file not found", "Error: file not found"),
)
def test_remove_color_codes(raw_text, clean_text):
    assert remove_color_codes(raw_text) == clean_text
```

Reply:
Github code editor sucks, so make sure to check formatting in your local editor.  But the point here is that you want your tests to be very simple and your test cases to be separate things wherever possible.  `pytest` has excellent features for parameterizing tests.